src/Spice86.Core/Emulator/Gdb/GdbBreakpointCommandParser.cs

```diff
@@ -0,0 +1,166 @@
+namespace Spice86.Core.Emulator.Gdb;
+
+using Spice86.Core.Emulator.CPU.CfgCpu.Ast.Parser;
+using Spice86.Core.Emulator.Memory;
+using Spice86.Core.Emulator.VM.Breakpoint;
+using Spice86.Shared.Emulator.VM.Breakpoint;
+using Spice86.Shared.Interfaces;
+using Spice86.Shared.Utils;
+using Serilog.Events;
+
+/// <summary>
+/// Parses GDB remote protocol breakpoint commands (Z packets) into structured data
+/// and creates Spice86 breakpoints from them.
+/// </summary>
+public class GdbBreakpointCommandParser {
+    private readonly ILoggerService _loggerService;
+    private readonly BreakpointConditionCompiler? _conditionCompiler;
+
+    public GdbBreakpointCommandParser(ILoggerService loggerService, BreakpointConditionCompiler? conditionCompiler = null) {
+        _loggerService = loggerService;
+        _conditionCompiler = conditionCompiler;
+    }
+
+    /// <summary>
+    /// Parses a GDB breakpoint command string into a structured GdbBreakpointCommand.
+    /// This method only parses the string format without interpretation or validation.
+    /// </summary>
+    /// <param name="command">The GDB Z command string (e.g., "0,1000,1" or "0,1000,1;X:ax==0x100").</param>
+    /// <returns>A parsed GdbBreakpointCommand, or null if parsing fails.</returns>
+    public GdbBreakpointCommand? Parse(string command) {
+        // Split on semicolon to separate base command from condition
+        string? conditionExpression = null;
+        string baseCommand = command;
+        
+        int semicolonIndex = command.IndexOf(';');
+        if (semicolonIndex >= 0) {
+            baseCommand = command.Substring(0, semicolonIndex);
+            string conditionPart = command.Substring(semicolonIndex + 1);
+            
+            // Parse condition part - format is "X:expression" or "cond:expression"
+            int colonIndex = conditionPart.IndexOf(':');
+            if (colonIndex >= 0) {
+                string condType = conditionPart.Substring(0, colonIndex);
+                if (condType.Equals("X", StringComparison.Ordinal) || 
+                    condType.Equals("cond", StringComparison.Ordinal)) {
+                    conditionExpression = conditionPart.Substring(colonIndex + 1);
+                    if (_loggerService.IsEnabled(LogEventLevel.Debug)) {
+                        _loggerService.Debug("Parsed GDB condition expression: {Condition}", conditionExpression);
+                    }
+                }
+            }
+        }
+        
+        // Parse base command: type,address,kind
```

Comment:
Why not use methods with proper names instead of low level codes with comments...